solver/src/liquidity/offchain_orderbook.rs

```diff
@@ -28,6 +33,23 @@ struct OrderSettlementHandler {
     order: Order,
 }
 
+fn inflight_order_filter(order: Order, inflight_trades: &HashSet<OrderUid>) -> Option<Order> {
+    // TODO - could model inflight_trades as HashMap<OrderUid, Vec<Trade>>
+    // https://github.com/gnosis/gp-v2-services/issues/673
+    if inflight_trades.contains(&order.order_meta_data.uid) {
+        return if order.order_creation.partially_fillable {
+            // TODO - driver logic for Partially Fillable Orders
+            // https://github.com/gnosis/gp-v2-services/issues/673
+            tracing::error!("filtering partially fillable inflight orders unimplemented.");
```

Comment:
nano nit: I wouldn't log an error in this case. The worst case that could happen is that our simulation fails due to order filled in which case it would already notify us.